command/pr.go

```diff
@@ -37,11 +37,13 @@ func init() {
 	prCmd.AddCommand(prReadyCmd)
 
 	prCmd.AddCommand(prListCmd)
+	prListCmd.Flags().BoolP("web", "w", false, "Open the browser to list the pull request(s)")
 	prListCmd.Flags().IntP("limit", "L", 30, "Maximum number of items to fetch")
 	prListCmd.Flags().StringP("state", "s", "open", "Filter by state: {open|closed|merged|all}")
 	prListCmd.Flags().StringP("base", "B", "", "Filter by base branch")
-	prListCmd.Flags().StringSliceP("label", "l", nil, "Filter by label")
+	prListCmd.Flags().StringSliceP("label", "l", nil, "Filter by labels")
 	prListCmd.Flags().StringP("assignee", "a", "", "Filter by assignee")
+	prListCmd.Flags().StringP("author", "A", "", "Filter by author")
```

Comment:
Let's only add `--author` once it's supported in both web and non-web mode https://github.com/cli/cli/pull/1102. Otherwise, people might be mislead to try it without `--web` and will be greeted with an error.